chess-AI/train.py

```diff
@@ -1,13 +1,13 @@
 from functools import reduce
-import random
-import time
 
 import chess
+import os
 import torch
 import numpy as np
 from torch.utils.data import DataLoader, TensorDataset
 
 from mcts import Mcts
+from ai_io import save_model, load_model
```

Comment:
I realize that PEP 8 says it's discouraged to have underscores in module names. Can you change it back to `checkpointing` sorry pls don't hate me :laughing: